lib/functions/image/partitioning.sh

```diff
@@ -83,13 +83,9 @@ PRE_PREPARE_PARTITIONS
 	local next=1
 	# Check if we need UEFI partition
 	if [[ $UEFISIZE -gt 0 ]]; then
-		if [[ "${IMAGE_PARTITION_TABLE}" == "gpt" ]]; then
-			local uefipart=15
-			# Check if we need BIOS partition
-			[[ $BIOSSIZE -gt 0 ]] && local biospart=14
-		else
-			local uefipart=$((next++))
-		fi
+		# Check if we need BIOS partition
+		[[ $BIOSSIZE -gt 0 ]] && local biospart=$((next++))
+		local uefipart=$((next++))
```

Comment:
This was the whole discussion on `sda15` and such. Let's merge as is and I'll handle in uefi-x86 later.